File: 코딩 기초 트레이닝/배열 만들기 5-p181912/배열 만들기 5.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181912

def solution(intStrs, k, s, l):
    answer = []

    for i in intStrs:
        i = i[s:s+l]
        if(int(i) > k):
            answer.append(int(i))

    return answer

File: 코딩 기초 트레이닝/배열 만들기 5-p181912/배열 만들기 5.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181912

function solution(intStrs, k, s, l) {
    const answer = [];

    intStrs.forEach((i) => {
        i = i.slice(s, s+l);
        if(Number(i) > k) answer.push(Number(i));
    });

    return answer;
}
